src/main/java/com/eternalcode/combat/CombatCommand.java

```diff
@@ -25,101 +30,167 @@ public class CombatCommand {
-        this.announcer.sendMessage(player, format);
+        this.announcer.sendMessage(sender, format);
     }
 
     @Execute(route = "tag", required = 2)
     @Permission("eternalcombat.tag")
-    public void tagMultiple(Player player, @Arg Player firstTarget, @Arg Player secondTarget) {
+    void tagMultiple(CommandSender sender, @Arg Player firstTarget, @Arg Player secondTarget) {
         Duration combatTime = this.config.settings.combatDuration;
         PluginConfig.Messages messages = this.config.messages;
-
-        UUID firstTargetUniqueId = firstTarget.getUniqueId();
-        UUID secondTargetUniqueId = secondTarget.getUniqueId();
-        UUID playerUniqueId = player.getUniqueId();
-
-        if (playerUniqueId.equals(firstTargetUniqueId) || playerUniqueId.equals(secondTargetUniqueId)) {
-            this.announcer.sendMessage(player, messages.admin.adminCannotTagSelf);
+        
+        if (sender.equals(firstTarget) || sender.equals(secondTarget)) {
+            this.announcer.sendMessage(sender, messages.admin.adminCannotTagSelf);
             return;
         }
-
-        this.fightManager.tag(firstTargetUniqueId, combatTime);
-        this.fightManager.tag(secondTargetUniqueId, combatTime);
+        
+        boolean isTaggedFirst = this.fightManager.tag(firstTarget.getUniqueId(), combatTime, CauseOfTag.COMMAND);
+        boolean isTaggedSecond = this.fightManager.tag(secondTarget.getUniqueId(), combatTime, CauseOfTag.COMMAND);
 
         Formatter formatter = new Formatter()
             .register("{FIRST_PLAYER}", firstTarget.getName())
             .register("{SECOND_PLAYER}", secondTarget.getName());
 
         String format = formatter.format(messages.admin.adminTagMultiplePlayers);
-        this.announcer.sendMessage(player, format);
 
-        this.announcer.sendMessage(firstTarget, messages.playerTagged);
-        this.announcer.sendMessage(secondTarget, messages.playerTagged);
+        if (isTaggedFirst) {
+            this.announcer.sendMessage(firstTarget, messages.playerTagged);
+        }
+        else {
+            this.announcer.sendMessage(sender, messages.admin.cannotTagPlayer);
+        }
+
+        if (isTaggedSecond) {
+            this.announcer.sendMessage(secondTarget, messages.playerTagged);
+        }
+        else {
+            this.announcer.sendMessage(sender, messages.admin.cannotTagPlayer);
+        }
+
+        if (isTaggedFirst || isTaggedSecond) {
+            this.announcer.sendMessage(sender, format);
+        }
+    }
+
+    @Async
+    @Execute(route = "reload")
+    @Permission("eternalcombat.reload")
+    void execute(CommandSender player) {
+        this.configService.reload();
+        this.announcer.sendMessage(player, this.config.messages.admin.reload);
     }
 
     @Execute(route = "untag", required = 1)
     @Permission("eternalcombat.untag")
-    void untag(Player player, @Arg Player target) {
+    void untag(Player sender, @Arg Player target) {
```

Comment:
```suggestion
    void untag(Player player, @Arg Player target) {
```

Reply:
?

Reply:
For me, sender means CommandSender, and player means Player.

Reply:
That was a followup to rollczi's comment on vc.
Sender means the player that sent a command, which is easier to read